docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -860,6 +860,8 @@ and/or history matching project.
 .. _custom_kw:
 .. topic:: CUSTOM_KW
 
+           `CUSTOM_KW` is deprecated.
```

Comment:
Already solved this here: https://github.com/equinor/ert/pull/954/commits/b14f9b7f2f02a4fa1f1c9c708c2acc2b3f3e5c9e
But I messed up my git history a bit, so I will just close mine

Reply:
Alright! Went a bit fast I suppose, didn't see that one